add invalidation types

YoanSallami opened the pull request:
add invalidation types to be used with waitforchanges()

severin-lemaignan commented:
If you do that (aliases for NEW, UPDATE, DELETE), would be nice to update `server.py` and `__init__.py` to replace all occurences of `gRPC.Invalidation.{...}` by the aliases. Otherwise it looks rather inconsistent.

YoanSallami commented:
I don't understand why travis failed... just one loading failed which is weird..

YoanSallami commented:
I've prepared to add two more properties for the nodes which could be useful.. "transparent" and "background" related to visibilities computation (see my issue).. when the PR will be closed, I will push the changes (their are in comment right now)

severin-lemaignan commented:
@YoanSallami travis failing on the mesh loading is an unrelated issue that I need to investigate (I can not reproduce it on my machine :-/), so no worries about that for now. I'm happy to merge the PR, but can you first remove all (commented out) changes related to the node properties? They really do not belong in here.

severin-lemaignan commented:
@YoanSallami cool! Tu me squashes les 4 commits ensemble, et je merge

Commit:
add invalidation alias to types

add invalidation types

try to make travis work and prepare for adding management for background and transparent object

remove comment

remove comments

fix error introduced in merging

severin-lemaignan commented:
@YoanSallami: to squash the commits, run `git rebase -i 4648a7bc8e600c8faca` (this correspond to the last commit before your PR), replace `pick` by `squash` for every line except the first one, save the file and exit the editor. Git will squash and ask you for a new commit message.

Finally, `git push --force` as you need to rewrite the history (which is fine in a PR)

YoanSallami commented:
I've made a mistake when merging, I didn't change all the Mesh to MeshData, travis didn't work so I will open a new one

severin-lemaignan commented:
Well, no need to create a new one, really! Keep pushing to this PR, that's fine! :-)

severin-lemaignan commented:
All right, can you squash and rebase your changes on `master`, and I merge them? :-)

YoanSallami commented:
done

severin-lemaignan commented:
Yeeeah! Merging it! Thanks!
